packages/web-ui/src/Alert/Alert.tsx

```diff
@@ -0,0 +1,182 @@
+      flexDirection: 'column',
+    },
+    '> button[data-dismiss]': {
+      alignSelf: 'flex-start',
+    },
+  },
+  '> :where(svg, [data-icon])': {
+    color: 'var(--alert-icon-color)',
+  },
+  '> :where(button)': {
+    color: 'var(--alert-icon-color)',
+    backgroundColor: 'transparent',
+    border: 'none',
+    cursor: 'pointer',
+    padding: 0,
+    alignSelf: 'center',
+    '&:focus-visible': {
+      outline: 'none',
+      borderRadius: px(4),
+      boxShadow: '0 0 0 2px var(--alert-focus-color)',
+    },
+  },
+  [COLORSCHEME_SELECTORS.info]: {
+    '--alert-background-color': colors.cyan50,
+    '--alert-icon-color': colors.cyan700,
+    '--alert-border-color': colors.cyan500,
+    '--alert-focus-color': colors.cyan700,
+  },
+  [COLORSCHEME_SELECTORS.success]: {
+    '--alert-background-color': colors.green50,
+    '--alert-icon-color': colors.green700,
+    '--alert-border-color': colors.green500,
+    '--alert-focus-color': colors.green700,
+  },
+  [COLORSCHEME_SELECTORS.warning]: {
+    '--alert-background-color': colors.gold50,
+    '--alert-icon-color': colors.gold700,
+    '--alert-border-color': colors.gold500,
+    '--alert-focus-color': colors.gold700,
+  },
+  [COLORSCHEME_SELECTORS.error]: {
+    '--alert-background-color': colors.red50,
+    '--alert-icon-color': colors.red700,
+    '--alert-border-color': colors.red500,
+    '--alert-focus-color': colors.red700,
+  },
+});
+
+const AlertIcon = ({ colorScheme }: { colorScheme: AlertProps['colorScheme'] }) => {
+  if (colorScheme === 'info') {
+    return <InformationMediumContainedIcon />;
+  }
+  if (colorScheme === 'success') {
+    return <TickMediumContainedIcon />;
+  }
+  return <WarningMediumContainedIcon />;
+};
+
+/**
+ * An `Alert` is a component that provides feedback messages to users.
+ *
+ * > This component does not need to be wrapped in a `ThemeProvider` and can be used standalone with other component libraries.
+ */
+export const Alert = React.forwardRef<
+  React.ElementRef<'div'>,
+  React.PropsWithChildren<PropsWithSx<AlertProps>>
+>(
+  (
+    {
+      className,
+      colorScheme = 'info',
+      direction = 'column',
+      children,
+      onDismiss,
+      title,
+      text,
+      linkText,
+      linkHref,
+      onClick,
+      ...props
+    },
+    ref
+  ) => {
+    const dataAttributeProps = {
+      [DATA_ATTRIBUTES.colorscheme]: colorScheme,
+      [DATA_ATTRIBUTES.orientation]: direction === 'column' ? 'vertical' : 'horizontal',
+    };
+    return (
+      <StyledElement
+        ref={ref}
+        className={clsx(componentClassName, className)}
+        role="alert" // Adding role for dynamic alerts
+        aria-live="assertive" // Making it announced immediately
+        aria-atomic="true" // Ensuring the entire alert is read as a whole
+        {...dataAttributeProps}
+        {...props}
+      >
+        <AlertIcon colorScheme={colorScheme} />
+        <div className={classNames.content}>
```

Comment:
I'm kinda unsure about this approach, though I don't know if I can fully articulate why. I would rather use DS components, like `Flex` than create multiple generic classnames which aren't related to some kind of state/variant or prop. I'd worry that a class such as `uwu-content` or similar becomes arbitrary and used across different components without any reason behind it. I also feel we should be using DS components as much as possible.

Reply:
Yeah fair point, I'll make the change 👍

Reply:
What about the unstyled button?